opencog/atoms/bind/DefineLink.cc

```diff
@@ -84,4 +78,23 @@ DefineLink::DefineLink(Link &l)
 	init(l.getOutgoingSet());
 }
 
+Handle DefineLink::get_definition(const Handle& alias) {
+	// Get all DefineLinks associated with that alias, beware that it
+	// will also return DefineLink with that alias as definition body.
+    IncomingSet defs = alias->getIncomingSetByType(DEFINE_LINK);
+
+    // Return the first (supposedly unique) definition
+    for (LinkPtr defl : defs) {
+	    DefineLinkPtr def(DefineLinkCast(defl->getHandle()));
+	    if (def->get_alias() == alias)
```

Comment:
well, nevermind, having this check is harmless, because stranger things have happened before.